alert-policies/nextcloud/HeapMemoryUsage.yml

```diff
@@ -0,0 +1,37 @@
+name: Heap Memory Usage
+
+description: |+
+  This alert is triggered  when the Heap Memory Usage is below 2 for atleast 5 minutes.
```

Comment:
"This alert is triggered when the Heap Memory Usage is below 2 for at least 5 minutes."